src/main/kotlin/no/nav/helse/prosessering/v1/PreprosseseringV1Service.kt

```diff
@@ -74,7 +76,20 @@ internal class PreprosseseringV1Service(
             dokumentUrls = komplettDokumentUrls.toList(),
             søkerAktørId = søkerAktørId
         )
+
+        if(melding.skalRegistrereAleneOmOmsorgenBarn()){
+            logger.info("Registrerer barn med alene om omsorgen.")
+            aleneOmOmsorgenProducer.leggPåKø(melding, metadata)
+        }
+
         melding.reportMetrics()
         return preprosessertArbeidstakerutbetalingMelding
     }
 }
+
+private fun ArbeidstakerutbetalingMelding.skalRegistrereAleneOmOmsorgenBarn(): Boolean {
```

Comment:
Skulle man ikke registrere barn som ikke var fra TPS?
Isåfall må man også sjekke om aktørId = null

Reply:
Selv om barnet kommer fra tps så vet man vel ikke nødvendigvis om man har alene om omsorg for det? Tror alle barn man sier man har alene omsorg for skal registreres? Litt usikker

Reply:
Ok. Godt mulig jeg blander noe med det Frode sa.

Reply:
Jeg kan dobbeltsjekke med Erik

Reply:
Siden vi gjør oppslag i dialogen og bruker oppgir aleneomsorg, så må det vel også bety at det skal registreres? Ellers hadde vi ikke trengt å gjøre oppslag?

Reply:
Det er nok riktig :)

Reply:
Hvis det oppdages under test at det ikke er riktig så er det kjapp fiks uansett